trace/Span.md

```diff
@@ -24,9 +27,54 @@ parent span id for the child Span.
 #### TraceOptions 
 Represents the options for a trace. It is represented as 1 byte (bitmap).
 
-##### Supported bits
+##### Supported Bits
```

Comment:
Mixed style.

Either capitalize all the words in the title or revert this line.

Reply:
done